zng/string.go

```diff
@@ -35,7 +36,8 @@ func DecodeString(zv zcode.Bytes) (string, error) {
 }
 
 func (t *TypeOfString) Parse(in []byte) (zcode.Bytes, error) {
-	return in, nil
+	normalized := norm.NFC.String(string(Unescape(in)))
+	return []byte(normalized), nil
```

Comment:
Use norm.NFC.Bytes to avoid conversions to and from string.

Reply:
Oh nice, I should have read the godoc more closely...